app/service/data_svc.py

```diff
@@ -49,8 +49,14 @@ def __init__(self, dao):
             await self.dao.create('core_parser', parser)
         return 'Saved ability: %s' % id
 
-    async def create_adversary(self, name, description, phases):
-        identifier = await self.dao.create('core_adversary', dict(name=name.lower(), description=description))
+    async def create_adversary(self, name, description, phases, locked, init=False):
+        id = await self.dao.get('core_adversary', dict(name=name.lower()))
```

Comment:
let's use a different variable name - maybe identifier or ident? id is a built-in python type so best not to shadow (rarely a negative consequence but better safe).

Reply:
I've switched them to ident.